ts-tests/.env.example

```
@@ -0,0 +1 @@
+NODE_ENV=<`development` or `staging` or `production`>
```

should this be aligned with the actual switch case?
e.g.
```suggestion
NODE_ENV=<`development` or `staging` or `test` or `ci`>
```

---

Exactly. I add different configurations to our repository. 

Imagine in staging environment, we only need to pull the `ts-tests` repo, and run the following commands

`
`